app/src/main/java/net/squanchy/speaker/widget/SpeakerDetailsLayout.kt

```diff
@@ -28,13 +25,13 @@ class SpeakerDetailsLayout(context: Context, attrs: AttributeSet) : LinearLayout
         speakerBio.text = parseHtml(speaker.bio)
     }
 
-    @TargetApi(Build.VERSION_CODES.N)
-    // The older fromHtml() is only called pre-24
+    @TargetApi(Build.VERSION_CODES.N)     // The older fromHtml() is only called pre-24
```

Comment:
this has now logic in a bunch of places, worth extracting a parseHtml top level function?

Reply:
Agree, but probably the end goal is to use Dante everywhere

Reply:
What Lorenzo says, aiming to replace them all. Not worth doing it now (it's 4 copies right now btw)